src/filter/gaussian.js

```diff
@@ -0,0 +1,46 @@
+import Image from '../image';
+import convolution from '../operator/convolution';
+
+export default function gaussianFilter(k) {
+
+	this.checkProcessable({
```

Comment:
checkProcessable requires as first parameter the name of the procedure. In this case 'gaussianFilter'